### DISCUSSION TITLE
Import master data into books

### DISCUSSION THREAD

**Discussion by magicrocks** [frappe/books#297] [General]
Is there any way to import the master data like customer, supplier into books?

**Comment by 18alantom** [comment]
Currently, there isn't a way. Since it's an important feature, we'll be adding in within the next two releases. Will update here when the updated version is released.

If you have any inputs or suggestions on this feature do mention it here we'll keep it in mind when developing the feature.

**Reply by 18alantom** [reply]
Added it into the roadmap. ([card](https://github.com/frappe/books/projects/1#card-75801929))

**Comment by 18alantom** [comment]
A data import tool has been added: https://github.com/frappe/books/pull/347
